Calibration/HcalAlCaRecoProducers/src/AlCaHBHEMuonProducer.cc

```diff
@@ -205,20 +216,23 @@ void AlCaHBHEMuonProducer::produce(edm::Event& iEvent, const edm::EventSetup& iS
   iEvent.put(outputMColl,          labelMuon_.label());
 }
 
-void AlCaHBHEMuonProducer::beginJob() { }
+void AlCaHBHEMuonProducer::endStream() {
+  globalCache()->nAll_  += nAll_;
+  globalCache()->nGood_ += nGood_;
+}
 
-void AlCaHBHEMuonProducer::endJob() {
-  edm::LogInfo("HcalHBHEMuon") << "Finds " << nGood << " good tracks in " 
-			       << nAll << " events from " << nRun << " runs";
+void AlCaHBHEMuonProducer::globalEndJob(const AlCaHBHEMuons::Counters* count) {
+  edm::LogInfo("HcalHBHEMuon") << "Finds " << count->nGood_ <<" good tracks in "
+			       << count->nAll_ << " events";
```

Comment:
Isn't this info available in the framework summary? It seems like a great deal of effort to make this thread safe when one could just get the same info elsewhere for free.